app/src/main/java/com/example/runningavater/initialFlow/InitialFlow7.kt

```diff
@@ -1,10 +1,207 @@
+                        .fillMaxWidth()
+                        .align(Alignment.TopCenter)
+                        .padding(0.dp, 50.dp, 0.dp, 0.dp),
+                horizontalAlignment = Alignment.CenterHorizontally,
+            ) {
+                Text(
+                    text = "あなたの名前とアイコン",
+                    color = Color.Black,
+                    fontSize = 32.sp,
+                )
+                Text(
+                    text = "を設定しよう！",
+                    color = Color.Black,
+                    fontSize = 32.sp,
+                )
+                if (imageUri != null) {
+                    AsyncImage(
+                        model = imageUri,
+                        contentDescription = null,
+                        modifier =
+                            Modifier
+                                .offset((0.dp), (35.dp))
+                                .size(220.dp)
+                                .clickable { launcher.launch("image/%") },
+                    )
+                } else {
+                    Image(
+                        painter = painterResource(id = R.drawable.initial_icon),
+                        contentDescription = "初期アイコン",
+                        modifier =
+                            Modifier
+                                .offset((0.dp), (35.dp))
+                                .size(220.dp)
+                                .clickable { launcher.launch("image/%") },
+                    )
+                }
+                TextField(
+                    value = text.value,
+                    onValueChange = { newValue -> text.value = newValue },
+                    placeholder = { Text(text = "名前をここに入力してね") },
+                    colors =
+                        TextFieldDefaults.textFieldColors(
+                            containerColor = GranulatedSugar,
+                        ),
+                    modifier =
+                        Modifier
+                            .padding(0.dp, 20.dp, 0.dp, 0.dp)
+                            .size(350.dp, 55.dp),
+                )
+                Text(
+                    text = "※名前とアイコンは",
+                    modifier =
+                        Modifier
+                            .align(Alignment.End)
+                            .padding(end = 100.dp),
+                )
+                Text(
+                    text = "後から変更できるよ",
+                    modifier =
+                        Modifier
+                            .align(Alignment.End)
+                            .padding(end = 20.dp),
+                )
+            }
+            Column(
+                modifier =
+                    Modifier
+                        .align(Alignment.BottomStart)
+                        .padding(20.dp, 0.dp, 20.dp, 90.dp),
+            ) {
+                NextButton(
+                    navController = navController,
+                    nextDestination = "initialFlow/8/${text.value}",
```

Comment:
[must]
ここは `${text.value}` を渡す必要はない。
（入力された文字が空文字だとクラッシュしてしまうため）

```suggestion
                    nextDestination = "initialFlow/8",
```

Reply:
[5686e53](https://github.com/NID-kt/toryumon-team-f/pull/115/commits/5686e534f264228301cf835465fda18ef67c16d9) で対応したっぽい